app/server/src/controllers/appsController.ts

```diff
@@ -3,51 +3,54 @@ import { AppLocals } from "../types";
 import { ErrorType } from "../errors/errorType";
 import { WodinWebError } from "../errors/wodinWebError";
 import { getSessionStore } from "../db/sessionStore";
+import asyncControllerHandler from "../errors/asyncControllerHandler";
 
 export class AppsController {
-    static getApp = async (req: Request, res: Response) => {
-        const {
-            configReader, appsPath, wodinConfig, wodinVersion
-        } = req.app.locals as AppLocals;
-        const { appName } = req.params;
-        // client can pass either sessionId or share (friendly id) parameter to identify session to reload
-        let sessionId = req.query.sessionId as string | undefined | null;
-        const { share } = req.query;
+    static getApp = async (req: Request, res: Response, next: Function) => {
```

Comment:
There's a `NextFunction` type in express (line 1) that you could use here (and throughout) but I am not sure if it matters really? (and perhaps complicates the tests where we pass in `jest.fn`)